### PR TITLE
Replace tf.keras with keras throughout KerasCV 

### LINKED ISSUE #1527: Replace `tf.keras` with `keras` throughout KerasCV
We should import from Keras directly instead of relying on `tf.keras` as a convention throughout our codebase.

### PR DESCRIPTION
**Pull request by soma2000-lang**
Resolves #1527 @ianstenbit 
I have reformatted my files using shell/format.sh  but dont know why the code format is still failing

### CODE REVIEW

**Comment by ianstenbit** [comment]
`I have reformatted my files using shell/format.sh but dont know why the code format is still failing`

Try `shell/lint.sh` -- this is probably where the issue is coming from. It'll let you know what needs to be changed.

Thanks for the PR!

**Comment by soma2000-lang** [comment]
@ianstenbit  I think I need to do "import tensorflow.keras as keras" instead of "import tensoflow as tf" to fix this issue.

**Comment by ID6109** [comment]
@soma2000-lang Try adding `import keras` to the imports wherever this issue arises. Removing `import tensorflow as tf` will lead to further issues while calling other functions in TensorFlow.

**Comment by ianstenbit** [comment]
I see #1531 is also open for this issue -- you two can decide amongst yourselves how to move forward, but let's close one of these PRs.

One nice thing you can do is that if one of you pushes a commit to the feature branch on the other's fork, then you'll both have commits in the PR and it will show co-authorship.

**Comment by ID6109** [comment]
@soma2000-lang I'm trying to push my commits onto your bula branch, but it's denying me permission.

**Comment by soma2000-lang** [comment]
@ID6109 Can you please try this https://tighten.com/insights/adding-commits-to-a-pull-request/

**Comment by ID6109** [comment]
> @ID6109 Can you please try this https://tighten.com/insights/adding-commits-to-a-pull-request/

The link is helpful only to maintainers.
I have made the commits on your branch bula, but I am unable to push it.

**Comment by soma2000-lang** [comment]
@ianstenbit It would be great if you can help us regarding this

**Comment by ianstenbit** [comment]
@soma2000-lang you can add @ID6109 as a contributor to your fork here:
https://github.com/soma2000-lang/keras-cv/settings/access.

Then @ID6109 will be able to push their changes to your dev branch.

**Comment by ID6109** [comment]
@soma2000-lang Do tell me when you do the needful.

**Comment by soma2000-lang** [comment]
@ID6109 I have given it. just confirm if you can push now

**Comment by ID6109** [comment]
remote: Permission to soma2000-lang/keras-cv.git denied to ID6109.
fatal: unable to access 'https://github.com/soma2000-lang/keras-cv.git/': The requested URL returned error: 403

I'm still getting the same error @soma2000-lang

**Comment by soma2000-lang** [comment]
@ID6109 have you got any invitation

**Comment by ID6109** [comment]
Just got one a minute ago. Hold on.

**Comment by ID6109** [comment]
@ianstenbit this should be all the changes necessary. Do let us know if we missed any.

File: keras_cv/models/segmentation/deeplab.py [hunk 1]
```diff
@@ -43,6 +43,18 @@ class DeepLabV3(keras.Model):
             mask based on feature from backbone and feature from decoder.
     """
 
+    def build(self, input_shape):
```
Comment (ianstenbit) [inline keras_cv/models/segmentation/deeplab.py:46]: Please revert this change in this file -- let's keep `build` where it is.

File: keras_cv/tools/checkpoint_conversion/ViT_weight_conversion.ipynb [hunk 2]
```diff
@@ -8,7 +8,10 @@
      "base_uri": "https://localhost:8080/"
     },
     "id": "k0_-IVVOk-9w",
-    "outputId": "4a4c48bf-9257-406b-9a04-2071549095d4"
+    "outputId": "4a4c48bf-9257-406b-9a04-2071549095d4",
```
Comment (ianstenbit) [inline keras_cv/tools/checkpoint_conversion/ViT_weight_conversion.ipynb:11]: Please revert changes to this file

**Review by ianstenbit — APPROVED** [review]
LG -- thank you!

**Comment by vijaybirju** [comment]
great job all of you. My request didn't get merged due to overlapping.
